### PR TITLE
DevHomeSDK: Add to AuthenticationState enum

### PR DESCRIPTION
**Pull request by ssparach**
## Summary of the pull request
The PR adds FixAccount to the existing AuthenticationState enum. 

## Detailed description of the pull request / Additional comments
The PR makes an additive DevHome SDK change. It adds to the AuthenticationState enum and updates the contract version accordingly. This enum value will be used for account remediation scenarios. 

## Validation steps performed
Local release build of Dev Home SDK. 

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Comment by ssparach** [comment]
Hi @EricJohnson327, the change in this PR successfully builds in release Dev Home SDK but encounters an error in the debug configuration. I do not expect that we would need it to build in the debug config but wanted to confirm here as well. Thanks!

**Comment by EricJohnson327** [comment]
> Hi @EricJohnson327, the change in this PR successfully builds in release Dev Home SDK but encounters an error in the debug configuration (locally). I do not expect that we would need it to build in the debug config but wanted to confirm here as well. Thanks!

We don't support building the sdk in debug.  I dont remember the specifics, but there was some kind of dependency/naming issue.  We didn't spend more time investigating since there's minimal code directly in the sdk, so debug really wasn't needed.

File: extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl [hunk 1]
```diff
@@ -144,7 +144,10 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     enum AuthenticationState {
         LoggedIn = 0,
-        LoggedOut = 1
+        LoggedOut = 1,
+
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)]
```
Comment (manodasanW) [inline extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl:149]: Are there any plans to group up contract version changes or are we not concerned about that right now and it is as needed?

**Review by manodasanW — APPROVED** [review]

**Review by dkbennett — APPROVED** [review]

**Review by vineeththomasalex — APPROVED** [review]

File: extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl [hunk 2]
```diff
@@ -144,7 +144,10 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     enum AuthenticationState {
         LoggedIn = 0,
-        LoggedOut = 1
+        LoggedOut = 1,
+
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)] 
+        FixAccount = 2
```
Comment (vineeththomasalex) [inline extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl:150]: Reconsider the name. Maybe "NeedsRepair" or something similar. "FixAccount" doesn't sound like a state. :)

**Comment by EricJohnson327** [comment]
@ssparach can you check comments and verify if this is good to go?

**Pull request closed without merging**